tensorboard/plugins/graph/README.md

```diff
@@ -0,0 +1,129 @@
+# Graph Dashboard
+
+The Graph dashboard provides a visual representation of a computation graph,
+designed for understanding ML models.
+
+## What are graphs?
+
+Programs written with TensorFlow and other frameworks may transform the source
+code into a computational graph first, before actually executing the operations.
+To illustrate the idea, this short snippet:
+
+```
+w = 2
+result = sqrt(w)
+```
+
+might be represented as a graph with
+
+-   3 nodes: w, sqrt, result
+-   2 edges: [w, sqrt], [sqrt, result]
+
+Frameworks such as TF2 allow users to manually annotate functions with the
+`@tf.function` decorator to indicate to TensorFlow that a graph should be
```

Comment:
"to indicate to" sounds some grammar error to me

Reply:
I think the grammar sounds natural to me, but it definitely sounds wordy.  Since this paragraph essentially outlines how graphs are created (next section), I've removed this paragraph and folded it into the ones below.